src/airdrop.ts

```diff
@@ -612,18 +618,16 @@ export class Airdrop {
     verifierState: PublicKey,
     recipient: PublicKey,
     authority: PublicKey,
-    proposal: PublicKey,
   ): Promise<web3.Transaction> {
     const verifierStateObj = await this.governanceVerifierProgram.account.verifierState.fetch(verifierState, 'single');
-    const { airdropState, governance } = verifierStateObj;
+    const {
+      airdropState, governance, eligibilityStart, eligibilityEnd,
```

Comment:
Include total amount? Might be helpful for a status dashboard later?

Reply:
Not useful later. This is only for making a transaction